src/js/tabs/onboard_logging.js

```diff
@@ -180,7 +180,12 @@ TABS.onboard_logging.initialize = function (callback) {
         
         // Offer a reasonable choice of logging rates (if people want weird steps they can use CLI)
         var loggingRates = [];
-        var pidRate = 8000 / PID_ADVANCED_CONFIG.gyro_sync_denom / PID_ADVANCED_CONFIG.pid_process_denom; 
+        var pidRateBase = 8000;
+
+        if (PID_ADVANCED_CONFIG.gyroUse32kHz !== 0) pidRateBase = 32000;
```

Comment:
Can you please use the form

```
if (PID_ADVANCED_CONFIG.gyroUse32kHz !== 0) {
    pidRateBase = 32000;
}
```

(All conditional statements should use blocks, and use separate lines, for maintainability's sake.)

Reply:
done and retested with 32k FC